mne/io/constants.py

```diff
@@ -845,6 +848,11 @@ def __setattr__(self, attr, val):  # noqa: D105
 # KRISS sensors
 #
 FIFF.FIFFV_COIL_KRISS_GRAD         = 9001
+#
+# Compumedics adult/pediatric gradiometer
+#
+FIFF.FIFFV_COIL_COMPUMEDICS_ADULT_GRAD      = 10001
```

Comment:
You will need to add these to the list of inconsistencies to ignore in `mne/io/tests/test_constants.py`. You can check locally that it fails now and passes eventually with `pytest mne/io/tests/test_constants.py`

Reply:
The test fails because the compumedics constants are not in the fiff_constants repo, right? How about I merge the compumedics specification to the fiff_constans repo first?

Reply:
That would work, too